pyfar/dsp/dsp.py

```diff
@@ -1647,9 +1657,9 @@ def normalize(signal, mode='time', normalize='max',
     # replace input with normalized_input
     normalized_signal = signal.copy()
     if mode == 'time':
-        normalized_signal.time = signal.time.copy() / values_norm * value
+        normalized_signal.time *= value / values_norm
     else:
-        normalized_signal.freq = signal.freq.copy() / values_norm * value
+        normalized_signal.freq *= value / values_norm
```

Comment:
Sorry I missed this before. We don't need the if/else here - multiplying with a scalar yields the same result in the time and frequency domain. It only was important that we calculated `value` and `values_norm` in the correct domain. This should yield the same result:

`narmalized_signal = signal * value / values_norm